css/structure/jquery.mobile.core.css

```diff
@@ -279,6 +279,7 @@ div.ui-mobile-viewport {
 	padding: 0;
 }
 .ui-btn-inline {
+	max-width:100% ;
```

Comment:
@arschmitz  Firstly i was thinking of adding a new class to selectmenu which restricts its width when it goes out of the page. This class would only be added to element having .ui-select class and only when it goes out of the page. Do you think this idea is better..??